src/Whisper/Whisper.tsx

```diff
@@ -2,7 +2,7 @@ import * as React from 'react';
 import PropTypes from 'prop-types';
 import OverlayTrigger from 'rsuite-utils/lib/Overlay/OverlayTrigger';
```

Comment:
For this PR, I prefer to put all the logic about RTL in `rsuite-utils/lib/Overlay/OverlayTrigger`.
Because, in addition to Whisper, all other Pickers use OverlayTrigger.

Reply:
@simonguo All pickers use `PickerToggleTrigger` which uses `OverlayTrigger`.

The `OverlayTrigger` exists in another repo, which has no `IntlContext`. Pass direction as a prop ? Or add a context consumer to `PickerToggleTrigger` ?

```js
const PickerToggleTrigger = React.forwardRef(
  (props: PickerToggleTriggerProps, ref: React.RefObject<any>) => {
    const { pickerProps, speaker, trigger = 'click', open, ...rest } = props;
    const placement = pickerProps.placement;

    return (
      <IntlContext.Consumer>
        {context => (
          <OverlayTrigger
            trigger={trigger}
            ref={ref}
            open={open}
            placement={placementPolyfill(placement, context?.rtl)}
            speaker={speaker}
            {..._.pick(pickerProps, PickerToggleTriggerProps)}
            {...rest}
          />
        )}
      </IntlContext.Consumer>
    );
  }
);
```

Reply:
Modifying `PickerToggleTrigger` should be a more reasonable solution.
Just follow yours.

Reply:
@simonguo done.